derive/src/lib.rs

```diff
@@ -187,6 +195,7 @@ enum Parameter {
     Crate(String),
     Deployments(Vec<Deployment>),
     Methods(Vec<Method>),
+    EventDerives(Vec<TypePath>),
```

Comment:
ubernit: I would move the `to_string()` to the parsing of the parameter types, just to keep it consistent with the other parameters, where the data is 1:1 what the builder is expecting.